src/styles/main.scss

```
@@ -1,3 +1,36 @@
-body {
+@import "../styles/utils/var.scss";
+@import "../styles/mixin/mixin.scss";
+@import "../styles/blocks/calendar.scss";
+
+* {
+  box-sizing: border-box;
+}
+
+.main-page {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.main-page__content {
   margin: 0;
+  min-width: $weekWidth;
+}
+
+@each $day, $number in $days-of-week {
+  .calendar--start-day--#{$day}:first-child {
+    margin-left: ($size + $cellGap) * $number;
+  }
+}
+
+@for $last-day from 28 through 31 {
+  .calendar--month-length--#{$last-day} 
+  .calendar__days:nth-child(n + #{$last-day + 1}) {
+    display: none;
+  }
+}
+
+.main-page__calend {
+  position: fixed;
+  left: 250px;
+  top: 100px;
```

```suggestion
margin: auto
```
And apply `height: 100vh; display: flex` for calendar's parent